src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -73,7 +73,7 @@ public boolean confirmInvoiceNoMatchOverride(final AccountsPayableDocument accou
 	protected boolean checkWireTransferTabState(final VendorCreditMemoDocument cmDocument) {

```

Comment:
Could this method be replaced with a call to the equivalent method on `PaymentSourcePreRulesService`?

I realize that the other service's method seems to be tailoring the question message for DV usage, despite base code also using that method for handling the equivalent PREQ logic. To keep the bulk of the logic consolidated in one place, I would suggest creating a `CuPaymentSourcePreRulesServiceImpl` subclass that adjusts the question message on a per-document basis, using a different config property for the base message if necessary. (I can look into preparing replication steps for the DV-tailored-message issue so that it can be reported to KualiCo.)

Thoughts?

Reply:
Just as a quick follow-up, I've created KFSPTS-33228 to get the DV-message-on-PREQ issue reported back to KualiCo.